TinyConstraints/Classes/Constraints.swift

```diff
@@ -31,13 +31,13 @@
 public typealias Constraint = NSLayoutConstraint
 public typealias Constraints = [Constraint]
 
-public enum ConstraintRelation: Int {
+public enum ConstraintRelation: Int, Sendable {
     case equal = 0
     case equalOrLess = -1
     case equalOrGreater = 1
 }
 
-public extension Collection where Iterator.Element == Constraint {
+@MainActor public extension Collection where Iterator.Element == Constraint {
```

Comment:
Strongly recommend the following instead:

```suggestion
@preconcurrency @MainActor 
public extension Collection where Iterator.Element == Constraint {
```

This makes it so that existing code that doesn't yet have strict concurrency enabled can use this library version without having to make changes to their code.